src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+import sys
+import random
+import paddle
+import paddle.fluid as fluid
+
+from save_model import saveModel
+
+def sum(name: str, shape: list, dtype, values):
+    paddle.enable_static()
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        if len(values) == 1:
+            inputs = fluid.layers.fill_constant(shape=shape, dtype=dtype, value=values[0])
+        else:
+            inputs = []
+            for i in values:
+                inputs.append(fluid.layers.fill_constant(shape=shape, dtype=dtype, value=i))
+        out = fluid.layers.sum(inputs)
```

Comment:
Hi, @OpenVINO-dev-contest  and @Patrick-Star125 : based on PaddlePaddle 2.1 'sum' Operator specification, I think we would better use 'paddle.sum()' function here.

Reply:
Thanks @OpenVINO-dev-contest  for correcting me, 'paddle.sum()'  will be set 'reduce_sum' type during Paddle export this Operator. let's still use   'fluid.layers.sum' here and 'default_opset::Add' to support this  'fluid.layers.sum' Operator.